magit-todos.el

```diff
@@ -267,7 +267,7 @@ This should generally be set automatically by customizing
   "Pull in repo TYPE quickly, with optional PATH."
   (let* ((cmd (format
                ;; "grep -r -b --with-filename 'TODO\\FIXME' %s"
-               "rg -b --with-filename --no-heading --no-line-number '%s' %s"
+               "rg -b --with-filename --no-heading --no-line-number '%s' %s &"
```

Comment:
Does that work?  The `&` should cause the command to return to Emacs before it completes, right?